en/sensor/px4flow.md

```diff
@@ -341,7 +341,7 @@ The PX4FLOW module outputs [MAVLink](https://mavlink.io/en/) packets on USB and
 
 The following list gives a short explanation of the current available parameters in the PX4FLOW firmware.
 
-> **Caution** Parameters are currently not written to ROM (reset at power loss).
+> **Caution** Parameters are currently not written to ROM (reset at power loss). But if you want to change them you can build your own version of firmware using instruction `PX4FLOW Developer Guide` mentioned  above. Code with parameters you can find [here](https://github.com/PX4/Flow/blob/master/src/modules/flow/settings.c).
```

Comment:
@szobov Which document is the `PX4FLOW Developer Guide` referenced?

Reply:
Actually, I don't know how to make self references inside one document, so it's this document: https://github.com/szobov/px4_user_guide/blob/c6671b794ffd1935ceccb3a6b7262d0c2f5c9516/en/sensor/px4flow.md#px4flow-developer-guide-developer_guide

Reply:
Thanks very much. FYI only, you can link to any anchor in the doc using the syntax like `[text displayed for the link](#anchor-name)`. The anchor name can come from the heading as one is created automatically (and you can copy from the table of contents) or you can define what you want the anchor to be - e.g. if you have a heading defined you can use curly braces to define the anchor: `## Some heading {#your_own_anchor}`

I've cross linked now.
https://github.com/PX4/px4_user_guide/commit/81fe8ec3f3e6059be355ac128ec8fd830d328fc0